test/rules/disallow-multiple-line-breaks.js

```diff
@@ -3,29 +3,37 @@ var assert = require('assert');
 
 describe('rules/disallow-multiple-line-breaks', function() {
     var checker;
+
     beforeEach(function() {
         checker = new Checker();
         checker.registerDefaultRules();
+        checker.configure({ disallowMultipleLineBreaks: true });
     });
+
     it('should report multiple line break', function() {
-        checker.configure({ disallowMultipleLineBreaks: true });
         assert(checker.checkString('x = 1;\n\n\ny = 2;').getErrorCount() === 1);
     });
+
     it('should not report single line break', function() {
-        checker.configure({ disallowMultipleLineBreaks: true });
         assert(checker.checkString('x = 1;\n\ny = 2').isEmpty());
     });
+
     it('should not report single line break when first line starts with #!', function() {
-        checker.configure({ disallowMultipleLineBreaks: true });
         assert(checker.checkString('#!/usr/bin/env node\nx = 1;').isEmpty());
         assert(checker.checkString('#!/usr/bin/env node\n\nx = 1;').isEmpty());
     });
+
+    it('should report error in the correct location when first line starts with #!', function() {
```

Comment:
This pull for the `string-checker` module, but test for the rule, could we add one to for the relevant module too?

Like here - https://github.com/jscs-dev/node-jscs/blob/30049c1da94b590b783c1ba948da1d639b8c06bc/test/string-checker.js#L38